pkg/machine/wsl/declares.go

```diff
@@ -12,6 +12,9 @@ const containersConf = `[containers]
 
 [engine]
 cgroup_manager = "cgroupfs"
+
+[network]
+firewall_driver="iptables"
```

Comment:
please add the issue link as comment here so we know why this is set right away.

Reply:
Thank you. I have fixed it.